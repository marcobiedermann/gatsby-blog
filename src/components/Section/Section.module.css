.section {
  padding-bottom: 3em;
  padding-top: 3em;
}
