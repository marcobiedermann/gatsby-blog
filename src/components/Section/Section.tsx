import React, { ReactNode } from 'react';
import * as styles from './Section.module.css';

export interface SectionProps {
  children: ReactNode;
}

function Section(props: SectionProps) {
  return <section className={styles.section} {...props} />;
}

export default Section;
