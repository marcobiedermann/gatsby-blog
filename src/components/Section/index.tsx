import React, { FC } from 'react';
import styles from './style.module.css';

const Section: FC = (props) => {
  return <section className={styles.section} {...props} />;
};

export default Section;
