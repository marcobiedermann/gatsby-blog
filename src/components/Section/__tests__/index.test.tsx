import { render } from '@testing-library/react';
import React from 'react';
import Section from '..';

describe('section component', () => {
  it('should render correctly', () => {
    const { container } = render(<Section>Section</Section>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
