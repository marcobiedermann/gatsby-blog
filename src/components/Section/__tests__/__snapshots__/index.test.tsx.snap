// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`section component should render correctly 1`] = `
<section>
  Section
</section>
`;
